src/sql/util.py

```diff
@@ -494,3 +499,83 @@ def get_default_configs(sql):
     del default_configs["parent"]
     del default_configs["config"]
     return default_configs
+
+
+def parse_connect_arg(connect_arg):
+    if not isinstance(connect_arg, str):
+        url = urlparse(str(connect_arg.url))
+    else:
+        url = urlparse(connect_arg)
+
+    username, password = url.username, url.password
+    host, port = url.hostname, url.port
+    database, drivername = url.path, url.scheme
+    if database.startswith("/"):
+        database = database[1:]
+
+    return {
+        "username": username,
+        "password": password,
+        "host": host,
+        "port": port,
+        "database": database,
+        "drivername": drivername,
+    }
+
+
+def get_config_file():
```

Comment:
this can be renamed `get_connections_config_file`

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/783/files#diff-97c7661adb8698813af7c891197f6961ea293d1d1b02be2d5928d07db813bbe2R29-R32)